Update M PLUS fonts

pkazmier opened the pull request:
The author of this wonderful font has made updates to the font as he is
working with Google for inclusion in fonts.google.com.  As part of that
work, the name has also changed from "M+" to "M PLUS".  This change now
contains the modern and new M PLUS fonts.

https://mplusfonts.github.io

Thanks to Coji Morishita for such a beautiful font.

pkazmier committed:
Update M PLUS fonts

The author of this wonderful font has made updates to the font as he is
working with Google for inclusion in fonts.google.com.  As part of that
work, the name has also changed from "M+" to "M PLUS".  This change now
contains the modern and new M PLUS fonts.

https://mplusfonts.github.io

Thanks to Coji Morishita for such a beautiful font.

braver commented:
Thanks! Looks like the one to include for us is “M PLUS Code”.  We only need the standard 4 variants. Does it come in woff format?

pkazmier commented:
I believe we want to include "M PLUS Code Latin" instead of "M PLUS Code".  The difference between the two is that "M PLUS Code" variant includes the Japanese characters which bloats the size of the font. I do not see any WOFF files available, which is why I included the TTF files for "M PLUS Code Latin".

Which variants would you like to keep? I only saw two used in the `fonts.less` file. I can remove all the others and only keep the "regular" and "bold" variants.

braver commented:
> The difference between the two is that "M PLUS Code" variant includes the Japanese characters which bloats the size of the font.

Right, makes sense. If that is the only difference I would still call it "M Plus Code" though, "Latin" is only a subset of the larger font. 

> Which variants would you like to keep

TTF is fine. I'd like to have regular, italic, bold and bold-italic if those are available.

pkazmier commented:
Unfortunately, M PLUS Code does not have a true italic, so I'll remove everything but regular and bold.

pkazmier committed:
Update LICENSE and remove extra variants

pkazmier commented:
All requested changes have been made.

braver commented on fonts/stylesheets/fonts.less:
```diff
@@ -256,8 +256,8 @@
-.font ( 'mplus', bold, normal, 'mplus/mplus-bold.woff' );
-.font ( 'mplus', normal, normal, 'mplus/mplus.woff' );
+.font ( 'mplus', bold, normal, 'mplus/MPLUSCodeLatin-Bold.ttf' );
+.font ( 'mplus', normal, normal, 'mplus/MPLUSCodeLatin-Regular.ttf' );
```
Ok, this is maybe an personal idiosyncrasy, but I like to standardize the filenames. It doesn’t really matter I guess, but I’m not ready to let that go yet 😅 so could you keep the original filenames (just ttf instead of woff?)

pkazmier replied:
Done.

I only changed the names as I was trying to be consistent with how the files came from the distribution site (fonts.google.com), but I understand your desire for consistency.

pkazmier committed:
Rename font files back to mplus*.ttf

When I submitted the PR, I used the file names from the font's
distribution site (fonts.google.com), but @braver has asked that the
original names are used instead for consistency.
